src/OrbitGl/TimeGraph.cpp

```diff
@@ -821,7 +821,7 @@ void TimeGraph::DrawIncompleteDataIntervals(Batcher& batcher, PickingMode pickin
 
 void TimeGraph::DrawTracks(Batcher& batcher, TextRenderer& text_renderer,
                            const DrawContext& draw_context) {
-  for (auto& track : track_manager_->GetVisibleTracks()) {
+  for (auto& track : track_manager_->GetTracksOnScreen()) {
```

Comment:
Don't use auto for simple types: https://google.github.io/styleguide/cppguide.html#Type_deduction